src/components/CvssCalculator.vue

```diff
@@ -249,7 +252,7 @@ function handlePaste(e: ClipboardEvent) {
     .vector-input {
       height: 100%;
       border-radius: 0;
-      padding-left: 30px;
+      padding-left: 12px;
```

Comment:
```suggestion
      padding-left: 0.75rem;
```

Reply:
Do you know this values from memory or is there any converter? 😆